app/controllers/token_auth/api/payloads_controller.rb

```diff
@@ -53,7 +53,7 @@ def find_token!
 
         return if @authentication_token
 
-        fail TokenAuth::Concerns::ApiResources::Api::Unauthorized
+        raise TokenAuth::Concerns::ApiResources::Api::Unauthorized
```

Comment:
Just a RuboCop reversal that I didn't feel like fighting with. They are identical methods from what I can tell.